scss/application.scss

```diff
@@ -144,8 +144,16 @@ body {
     margin: 60px auto;
     display: grid;
     justify-content: center;
+    justify-items: center;
     grid-template-columns: repeat(3, max-content);
   }
+
+  @media (min-width: 600px) and (max-width: 900px) {
```

Comment:
There's no need for the `min-width`, move the classes inside this media query to the bottom with the other :smile: